src/EntityFramework/Core/EntityClient/DbParameterCollectionHelper.cs

```diff
@@ -292,6 +292,50 @@ private static int IndexOf(IEnumerable items, string parameterName)
             return -1;
         }
 
+        /// <inhertidoc />
```

Comment:
Probably right, the others already in this class were the same, so I stuck with it.